cypress/e2e/tests/migration/migration-waves/pagination.test.ts

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    selectItemsPerPage,
+    goToPage,
+    goToLastPage,
+    deleteAllMigrationWaves,
+    createMultipleMigrationWaves,
+} from "../../../../utils/utils";
+import * as commonView from "../../../views/common.view";
+import { MigrationWave } from "../../../models/migration/migration-waves/migration-wave";
+
+let migrationWavesList: MigrationWave[] = [];
+//Automates Polarion TC 357
+describe(["@tier3"], "Application inventory pagination validations", function () {
+    before("Login and Create Test Data", function () {
+        // Perform login
+        login();
+        deleteAllMigrationWaves();
+        // Navigate to Migration Waves tab, delete all and create 11 migration waves
+        migrationWavesList = createMultipleMigrationWaves(11);
+    });
+
+    after("Perform test data clean up", function () {
+        // Delete the Migration Waves created before the tests
+        deleteAllMigrationWaves();
+    });
```

Comment:
This block should be at the end of the file to follow a logical lifecycle 

`BeforeAll -> BeforeEach -> Tests -> AfterEach -> AfterAll`